config/config.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+return [
+
+    /*
+   |--------------------------------------------------------------------------
+   | Preview Mode
+   |--------------------------------------------------------------------------
+   |
+   | This option is available to allow users to control if the preview tab
+   | should be shown or not.
+   |
+   */
+
+    'preview_enabled' => true
```

Comment:
Still need to remove this @jaxwilko

Reply:
I'd rather leave it as a simple way for users to disable the preview if they want to, otherwise they'll have to register some horrible events due to the way preview is added in the `backend.form.extendFieldsBefore`

Reply:
I disagree :P 

```php
// Add the Preview tab as the last tab with a event priority that still
// allows other plugins to change it if desired.
```
So just the below will allow you to disable it:
```php
        Event::listen('backend.form.extendFieldsBefore', function ($formWidget) {
            if (
                $formWidget->isNested
                || $formWidget->isNested
                || !($formWidget->getController() instanceof Index)
                || !($formWidget->model instanceof StaticPage)
            ) {
                return;
            }

            unset($formWidget->fields['_preview'];
            unset($formWidget->tabs['fields']['_preview'];
            unset($formWidget->secondaryTabs['fields']['_preview'];
        }, PHP_INT_MIN);
```

Reply:
How is this less horrible than:

```php
 'preview_enabled' => false
 ```
 
 :joy:

Reply:
Because it shouldn't be common for someone to want to disable the preview and if they do then they should have to do it in code 😜

Reply:
Wouldn't they just disable the plugin if they wanted to disable the preview? :P